H.Ws/SQL

```diff
@@ -0,0 +1,36 @@
+show databases;
+use baseball;
+
+show tables;
+
+
+#Historic Average 
+
+SELECT (SUM(Hit)/SUM(atBat)) FROM batter_counts AS Batting_Average
+GROUP BY batter;
+
+
+#Create table with date
+
+CREATE TABLE IF NOT EXISTS Averages_table
```

Comment:
I like the CREATE TABLE IF NOT EXISTS line in your code. I use "Drop if exists" and then another line for "create table" I am going to be stealing this thank you!